### PR TITLE
Mount and umount

### PR DESCRIPTION
**Pull request by amercer1**
Unmount and mount are one of very few remaining Libcloud Deployments for atmosphere, but ansible is much better equipped to answer these types of questions.

### CODE REVIEW

File: ansible/roles/unmount-volume/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version < '5')
+
+- name: register mountpoint and fstype
+  shell: grep "^/dev/vdc " /proc/mounts | cut -d ' ' -f 2,3 | awk '{ printf "[\"%s\",\"%s\"]\n", $1, $2 }' 
+  register: result
+
+- debug: Print out mountpoints for the device {{ src }}
+  msg="{{ (item |from_json )[0] }} {{ src }} {{ (item |from_json )[1] }}" 
+  with_items:
+    - "{{ result.stdout_lines }}"
```
Comment (steve-gregory) [inline ansible/roles/unmount-volume/tasks/main.yml]: Wondering why you're calling with items and calling explicitly for index [0]?

File: ansible/roles/unmount-volume/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+# ansible-playbook playbooks/utils/unmount_volume.yml -i hosts --limit "vm0" -e "{ 'src':'/dev/vdc' }"
+
+
+- set_fact: ansible_python_interpreter=/usr/bin/python2.6
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version < '5')
+
+- name: register mountpoint and fstype
+  shell: grep "^/dev/vdc " /proc/mounts | cut -d ' ' -f 2,3 | awk '{ printf "[\"%s\",\"%s\"]\n", $1, $2 }'
```
Comment (steve-gregory) [inline ansible/roles/unmount-volume/tasks/main.yml]: Hard coded /dev/vdc

Comment (amercer1) [inline ansible/roles/unmount-volume/tasks/main.yml]: Haha that will be fixed. Whoops

File: ansible/roles/unmount-volume/tasks/main.yml (hunk 1 above)
Comment (amercer1) [inline ansible/roles/unmount-volume/tasks/main.yml]: The `with_items` is to loop through all of the mountpoints that device has been attached to.
```
root@vm64-225:/# grep "^/dev/vdc " /proc/mounts | cut -d ' ' -f 2,3 | awk '{ printf "[\"%s\",\"%s\"]\n", $1, $2 }'                                                                 
["/vol2","ext3"]
["/vol1","ext3"]
```

And the `[0]` is to grab out the mountpoint data required. `[1]` is for fstype.

Comment (steve-gregory) [inline ansible/roles/unmount-volume/tasks/main.yml]: Aha! Makes sense to me! 👍

**Comment by steve-gregory** [comment]
The changes look great, thanks @amercer1 ! I see that you've got some comments in the role about how to call the playbook, can you please convert that to a "usage" block to the playbooks (when called with no args) to help users who come across them better understand how they are called?

Thanks 👍

**Comment by steve-gregory** [comment]
:shipit:

**Pull request merged**